packages/cli/src/commands/config/print.ts

```diff
@@ -16,16 +16,41 @@
 
 import { Command } from 'commander';
 import { stringify as stringifyYaml } from 'yaml';
+import { ConfigReader } from '@backstage/config';
 import { loadCliConfig } from '../../lib/config';
+import cloneDeepWith from 'lodash/cloneDeepWith';
 
 export default async (cmd: Command) => {
-  const { config } = await loadCliConfig(cmd.config, cmd.withSecrets ?? false);
+  const { schema, appConfigs } = await loadCliConfig(cmd.config);
+  let data;
 
-  const flatConfig = config.get();
+  if (cmd.frontend) {
```

Comment:
I wonder if we could break this out into another function that just has `return` for the conditional rather than using `let` and reassigning. Might be a little cleaner?